common/src/main/java/com/wynntils/features/user/tooltips/ItemCompareFeature.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.tooltips;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyHolder;
+import com.wynntils.mc.event.InventoryKeyPressEvent;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.mc.event.ScreenClosedEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.objects.CommonColors;
+import com.wynntils.wc.custom.item.GearItemStack;
+import java.util.List;
+import java.util.Optional;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@FeatureInfo(stability = FeatureInfo.Stability.STABLE, category = "Item Tooltips")
+public class ItemCompareFeature extends UserFeature {
+    @RegisterKeyBind
+    private final KeyHolder toggleCompareModeKeybind =
+            new KeyHolder("Compare mode", GLFW.GLFW_KEY_SPACE, "Wynntils", true, () -> {});
+
+    @RegisterKeyBind
+    private final KeyHolder compareSelectKeybind =
+            new KeyHolder("Select for comparing", GLFW.GLFW_KEY_C, "Wynntils", true, () -> {});
+
+    private static GearItemStack COMPARED_ITEM = null;
+    private static boolean COMPARE_MODE_ON = false;
+
+    @SubscribeEvent
+    public void onRenderSlot(SlotRenderEvent.Pre event) {
+        Slot slot = event.getSlot();
+        if (slot.getItem() == COMPARED_ITEM) {
+            RenderUtils.drawArc(CommonColors.LIGHT_BLUE, slot.x, slot.y, 200, 1, 6, 8);
+        }
```

Comment:
Is this rendered in the example pictures?

Reply:
![image](https://user-images.githubusercontent.com/49001742/183735162-55d39fcb-f16b-4f5e-bd22-371ca16f7b75.png)